report/src/sections/01-introduction.tex

```diff
@@ -40,10 +42,19 @@ \subsection{Non-terminating Analysis}\label{subsec:non-terminating-analysis}
     \label{lst:motivate-program}
 \end{listing}
 
+\begin{listing}
+    \begin{minted}{sql}
+    INSERT INTO account VALUES ("name", 1);
+    \end{minted}
+    \caption{A simple SQL statement inserting a tuple into the account table.}
+    \label{lst:motivate-sql-insert}
+\end{listing}
+
+
+The database context would constantly change under Cortesi and Halder's framework, potentially leading to analysis termination issues.
+However, there is a solution.
+Extending the analysis to include abstract tuples where the values are abstract, the analysis can terminate even for the program in \autoref{lst:motivate-program}.
```

Comment:
Saying that both the tuples and the multiplicity of tuples in the table is abstracted over would be more correct.